src/Language/Pirouette/QuasiQuoter/Syntax.hs

```diff
@@ -121,16 +120,11 @@ deriving instance
 
 -- * Parsers
 
--- | Parses a program, consisting in a map of declarations and one distinguished
---  function called main.
+-- | A program consists in a number of declarations
```

Comment:
```suggestion
-- | A program consists of a number of declarations
```
or
```suggestion
-- | A program is some set of declarations
```
(the latter also emphasizes unordered structure at this point)

Reply:
I like the second more! Will update! Thanks